packages/muelu/example/advanced/multiplesolve/ReuseSequence.cpp

```diff
@@ -235,9 +236,9 @@ Teuchos::RCP<Matrix> BuildMatrix(bool is3D, const Tensor<Scalar>& tensor, Teucho
   std::vector<GO> inds(nnz);
   std::vector<SC> vals(nnz);
 
-  ArrayRCP<const SC> x = coords->getData(0);
-  ArrayRCP<const SC> y = coords->getData(1);
-  ArrayRCP<const SC> z = (is3D ? coords->getData(2) : Teuchos::null);
+  ArrayRCP<const Real> x = coords->getData(0);
```

Comment:
I get the type `Real` from `MultiVectorReal`. So if we decided at some point to change the instantiation of `MultiVectorReal`, this would also switch over this type.

Reply:
Technically the coordinates can be `float`, `double` or `long double` but not the associated `std::complex<>` type so it makes sense to provide a `REAL` type that can be used for these distinctions. I am guessing that when someone instantiates on `float` this type is implicitly casted as `double`? This might cause problem for `long double` though?

Reply:
@lucbv Perhaps, but MueLu itself requires Coordinates to be double.  I can't imagine a case in the driver where casting from `long` to `double` would be problematic.